docs/HowTo/Use-Privacy/Sign-Privacy-Marker-Transactions.md

```diff
@@ -0,0 +1,14 @@
+description: How to sign a Privacy Marker Transaction
+<!--- END of page meta data -->
+
+# Signing Private Marker Transactions
+
+Private Marker Transactions can be signed with a random key or private key. To sign Private Marker Transactions with a private key, use [`--privacy-marker-transaction-signing-key-file`](../../Reference/Pantheon-CLI/Pantheon-CLI-Syntax.md#privacy-marker-transaction-signing-key-file) when starting Pantheon.
+
+!!! important
+    In networks where gas is paid, a funded private key must be specified.
+    
+In [free gas networks](../../HowTo/Configure-Pantheon/FreeGas.md), exclude `--privacy-marker-transaction-signing-key-file` to sign the Private Marker Transaction with a random key to add additional anonymity.
```

Comment:
I'd probably reword this slightly to address the following: 
- it's not clear from reading this alone that `--privacy-marker-transaction-signing-key-file` is a command line option rather than something specified at the time of creating the private transaction 
- same point as above, if the `---privacy-marker-transaction-signing-key-file` CLI option is specified, each PMT is signed with a different random key